backend/vertical-healthcare_ch/forum_datenaustausch_ch.invoice_440_response/src/test/resources/Cancelation_KV_12345.xml

```diff
@@ -8,7 +8,7 @@
 		</invoice:transport>
 	</invoice:processing>
 	<invoice:payload copy="false" credit_advice="false" response_timestamp="1537300879" storno="false" type="invoice">
-		<invoice:invoice request_date="2018-09-03T00:00:00" request_id="KV_12345" request_timestamp="1535971639" />
+		<invoice:invoice request_date="2018-09-02T22:00:00" request_id="KV_12345" request_timestamp="1537300879" />
```

Comment:
```suggestion
		<invoice:invoice request_date="2018-09-03T00:00:00" request_id="KV_12345" request_timestamp="1537300879" />
```

This is causing JUnits to fail. see https://metasfresh.testspace.com/spaces/228783/current/330333/junit/backend/vertical-healthcare_ch/forum_datenaustausch_ch.invoice_440_response